.github/renovate.json

```diff
@@ -34,6 +34,10 @@
     {
       "matchPackageNames": ["react-textarea-autosize"],
       "allowedVersions": "<8.4.1"
+    },
+    {
+      "matchPackageNames": ["react-from-dom"],
```

Comment:
We've already had to downgrade this dependency last time because the new one conflicts with our current Storybook setup so I'm adding this rule here to avoid hitting this stone again in the future.